packages/model-registry/package.json

```diff
@@ -45,8 +45,10 @@
     }
   },
   "exports": {
-    "./extensions": "./extensions.ts"
-  },
+    "./extensions": "./extensions.ts",
+    "./model-catalog-deploy": {
+      "types": "./upstream/frontend/src/odh/extension-points/model-catalog-deploy.ts"
+    }  },
```

Comment:
Hm, we don't have linters on these package.json files I guess - the last `},` here should be on its own line